linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -458,14 +461,29 @@ pipeline {
                     "${LISAV2_PARAMS}" +
                     " -CustomKernel 'localfile:./scripts/package_building/${env.BUILD_NUMBER}-${env.BRANCH_NAME}-${env.KERNEL_ARTIFACTS_PATH}/*/${env.PACKAGE_TYPE}/*.${env.PACKAGE_TYPE}'" +
                     " -OsVHD '${HYPERV_VHD_PATH}'" +
-                    " -XMLSecretFile '${HyperV_Secrets_File}'"
+                    " -XMLSecretFile '${HyperV_Secrets_File}'" +
+                    " -ExitWithZero"
                 )
             }
           }
           post {
             always {
               junit "Report\\*-junit.xml"
               archiveArtifacts "TestResults\\**\\*"
+              script {
+                    def returnValues = powershell returnStdout: true, script: '''$content = [xml](Get-Content ".\\Report\\*-junit.xml")
+                        $fail_case = [int]($content.testsuites.testsuite.failures)
```

Comment:
you still have variables which use camel_case instead of camelCase, rename all of them